pykickstart/sections.py

```diff
@@ -849,3 +849,82 @@ def _warn_alias_future_deprecation(self, option, new_option, lineno):
         ) % {"option": option, "lineno": lineno, "new_option": new_option},
             KickstartDeprecationWarning
         )
+
+class CertificateSection(Section):
+    sectionOpen = "%certificate"
+    _title = "Certificate"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._certificate = {
+            "name": None,
+            "path": "/etc/pki/ca-trust/source/anchors/",
+            "cert": []
+        }
+
+    def _getParser(self):
+        op = KSOptionParser(prog=self.sectionOpen, description="""
+                            The %certificate section is used to specify a
+                            certificate to be installed on the system.
+
+                            Example:
+
+                            %certificate --name custom_certificate.pem --path /etc/pki/ca-trust/source/anchors/
+                            -----BEGIN CERTIFICATE-----
+                            MIIDkDCCAnigAwIBAgIUFfTKU02DB4Nz4u3pRk1ShpRvn0AwDQYJKoZIhvcNAQEL
+                            BQAwVTELMAkGA1UEBhMCVVMxGzAZBgNVBAgTElNvbWUgU3RhdGUgb3IgUHJvdmlu
+                            Y2UxEjAQBgNVBAcTCVJvYWxkIE9uZTEVMBMGA1UEChMMRXhhbXBsZSBPcmcgTjAw
+                            HhcNMjEwNzI2MDg0NDIxWhcNMjIwNzI2MDg0NDIxWjBVMQswCQYDVQQGEwJVUzEb
+                            MBkGA1UECBMSU29tZSBTdGF0ZSBvciBQcm92aW5jZTESMBAGA1UEBxMJUm9hbGQg
+                            T25lMRUwEwYDVQQKEwxFeGFtcGxlIE9yZyBObzAwggEiMA0GCSqGSIb3DQEBAQUA
+                            A4IBDwAwggEKAoIBAQC4/SDsn8RQk0Euh6ZTKq5/Mz34K6QlnrxmAF7B8QGbDiK6
+                            ...
+                            -----END CERTIFICATE-----
+                            %end
+                            """, version=F42)
+
+        op.add_argument("--name", dest="name", required=True, version=F42,
```

Comment:
Would it make sense to rename `--name` to `--filename`? It would be easier to understand the purpose of the option and avoid the users to use it as a description of the certificate, for example: `--name="Cert for my HTTPS server"`.